automation/terraform/modules/kubernetes/testnet/variables.tf

```diff
@@ -203,15 +203,15 @@ variable "seed_external_port" {
 variable "seed_configs" {
   type = list(
     object({
-      name               = string,
-      class              = string,
-      libp2p_secret      = string,
+      name             = string,
+      class            = string,
+      libp2p_secret    = string,
       libp2p_secret_pw = string
       # external_port      = number,
```

Comment:
Do you have any plans to remove those comments. They are located not only in this file but across all terraform sources? I wonder if they were only leftovers ?